From e8dd5ecb2edaa058b1ec24ee8c7e86f5e672aa20 Mon Sep 17 00:00:00 2001
From: xujie2414 <xujie2414@163.com>
Date: Sat, 23 Sep 2023 21:51:54 +0800
Subject: [PATCH] modify patch

---
 README.md | 1 +
 1 file changed, 1 insertion(+)

diff --git a/README.md b/README.md
index 5aea154..ea0ba47 100644
--- a/README.md
+++ b/README.md
@@ -5,4 +5,5 @@ sudo miniocm 1111
 sudo minicom 2222
 33333333333
 44444444444
+patch test
 '''
-- 
2.17.1

